README.md

```diff
@@ -183,6 +183,7 @@ The new Firefox DevTools are powerful, flexible, and best of all, hackable. This
 - [Emacs](https://www.gnu.org/software/emacs/)
 - [Vim](https://www.vim.org/) - edit text at the speed of thought ;)
 - [Visual Studio Code](https://code.visualstudio.com/) - great for front end development, Visual Studio Code Editor by Microsoft
+- [Visual Studio Code Tips and Tricks] (https://code.visualstudio.com/shortcuts/keyboard-shortcuts-macos.pdf) - handy cheat sheet for VSCode shortcuts and tricks. 
```

Comment:
```suggestion
- [Visual Studio Code Tips and Tricks](https://code.visualstudio.com/shortcuts/keyboard-shortcuts-macos.pdf) - handy cheat sheet for VSCode shortcuts and tricks. 
```

Reply:
need to remove the space in order for the link to work 👍

Reply:
ok thanks!